src/microbe_stage/MicrobeHUD.cs

```diff
@@ -51,9 +51,13 @@ public class MicrobeHUD : CreatureStageHUDBase<MicrobeStage>
 
     private ProgressBar ingestedMatterBar = null!;
 
+    private Timer becomeMulticellularTutorialTimer = null!;
```

Comment:
This is not the way I've done the other timed tutorials. Please read through `MicrobePressEditorButton` class and implement this feature the same way.